packages/cra-template/package.json

```
@@ -8,8 +8,7 @@
     "directory": "packages/cra-template"
   },
   "scripts": {
-    "lint": "eslint --ext .js,.ts,.tsx .",
-    "test": "node -r @fluentui/scripts/babel/register scripts/test.ts"
+    "lint": "eslint --ext .js,.ts,.tsx ."
```

It's fine to disable it to unblock release but we should not be disabling this in master

```suggestion
    "lint": "eslint --ext .js,.ts,.tsx .",
    "test": "node -r @fluentui/scripts/babel/register scripts/test.ts"
```